Tests/TuistCoreTests/Graph/GraphTests.swift

```diff
@@ -780,6 +805,64 @@ final class GraphTests: TuistUnitTestCase {
         XCTAssertTrue(result.isEmpty)
     }
 
+    func test_runPathSearchPaths() throws {
+        // Given
+        let precompiledNode = mockDynamicFrameworkNode(at: AbsolutePath("/test/test.framework"))
+        let precompiledNodeB = mockDynamicFrameworkNode(at: AbsolutePath("/test/test.framework"))
+        let unitTests = Target.test(name: "AppUnitTests", product: .unitTests)
+        let project = Project.test(path: "/path/a")
+
+        let unitTestsNode = TargetNode(project: project, target: unitTests, dependencies: [precompiledNode, precompiledNodeB])
+
+        let cache = GraphLoaderCache()
+        cache.add(project: project)
+        cache.add(precompiledNode: precompiledNode)
+        cache.add(precompiledNode: precompiledNodeB)
+        cache.add(targetNode: unitTestsNode)
+
+        let graph = Graph(name: "Graph",
+                          entryPath: project.path,
+                          cache: cache,
+                          entryNodes: [unitTestsNode])
+
+        // When
+        let got = graph.runPathSearchPaths(path: project.path, name: unitTests.name)
+
+        // Then
+        XCTAssertEqual(
+            got,
+            [AbsolutePath("/path/to")]
```

Comment:
is this correct? I would have thought `/test`? 

ah

```swift
 private func mockDynamicFrameworkNode(at path: AbsolutePath) -> FrameworkNode {
        let precompiledNode = FrameworkNode.test() <--- we're missing passing the path here
```

and it has a default value of `/path/to/Framework.framework`

```swift
public extension FrameworkNode {
    static func test(path: AbsolutePath = "/path/to/Framework.framework",

```